javascript/javascript1/week2/flight.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * 
+ * @param {String} firstName 
+ * @param {String} surName 
+ * @param {Boolean} userFormalName 
+ 
+ */
+
+
+//created a function to get the full name of a person
+const getFullName=(firstName,lastName, userFormalName)=>{
+
+  if(!firstName && !lastName){
```

Comment:
Use .trim() to remove unnecessary spaces. If the user enters only spaces ('   '), the function considers it valid.